boards/arm/nrf54h20pdk_nrf54h20/nrf54h20pdk_nrf54h20_cpuapp.dts

```diff
@@ -97,6 +97,11 @@
 	status = "okay";
 };
 
+&cpuppr_vpr {
+	source-memory = <&cpuppr_code_partition>;
+	execution-memory = <&cpuppr_ram3x_region>;
+};
```

Comment:
Maybe this could be moved to the `nordic-ppr` snippet now?

Reply:
We can, but isn't that the default configuration based on the `memory_map` file?

Reply:
also note that the snipper thing is an experiment, not sure if a good idea :smile:

Reply:
> We can, but isn't that the default configuration based on the `memory_map` file?

It is, and the snippet is already tied to that memory map. But after mulling this over, I think it can stay like this.

> also note that the snippet thing is an experiment, not sure if a good idea 😄

It'll definitely come in handy :) Plus, there's room for it to evolve over time.